src/components/css/input.css

```diff
@@ -0,0 +1,3 @@
+.invalid {
```

Comment:
I think this style is not used anymore. We're not giving a visual indicator that the value is not valid.

Reply:
right